octobot_trading/exchange_data/prices/price_events_manager.py

```diff
@@ -34,6 +34,7 @@ class PriceEventsManager:
     def __init__(self):
         self.logger = logging.get_logger(self.__class__.__name__)
         self.events = []
+        self._last_recent_trades = []
```

Comment:
I don't think it's a good idea to store recent trades because the price is calculated from many sources not only recent trades. Price Manager triés to abstract the way we manage price, I think we shouldn't break this abstraction. To obtain the same goal, we should store the last X prices within a predefined time (it will contain recent trades an other sources prices) What do you think?

Reply:
Agree, I'll draft something

Reply:
idea pushed :)